modules/literature/src/main/resources/schemas/neurosciencegraph/literature/scholarlyarticle/v0.1.0.json

```diff
@@ -0,0 +1,115 @@
+                                }
+                            ],
+                            "minCount": 1
+                        },
+                        {
+                            "path": "schema:datePublished",
+                            "name": "publicationDate",
+                            "description": "Date of first publication",
+                            "datatype": "xsd:date",
+                            "minCount": 1,
+                            "maxCount": 1
+                        },
+                        {
+                            "path": "schema:isPartOf",
+                            "name": "volume",
+                            "description": "The volume in which this article appears",
+                            "class": "schema:PublicationVolume",
+                            "node": "{{base}}/schemas/neurosciencegraph/literature/publicationvolume/v0.1.0/shapes/PublicationVolumeShape",
+                            "minCount": 1,
+                            "maxCount": 1
+                        },
+                        {
+                            "path": "schema:url",
+                            "name": "url",
+                            "description": "URL at which the full text or abstract of the scholarly article can be obtained.",
+                            "nodeKind": "sh:IRI"
+                        },
+                        {
+                            "path": "schema:keywords",
+                            "name": "keywords",
+                            "description": "Keywords or tags used to describe this article. Multiple entries in a keywords list are typically delimited by commas.",
+                            "datatype": "xsd:string"
+                        },
+                        {
+                            "path": "schema:license",
+                            "name": "license",
+                            "description": "A license document that applies to this article, typically indicated by URL.",
+                            "nodeKind": "sh:IRI"
+                        },
+                        {
+                            "path": "schema:pagination",
+                            "name": "pagination",
+                            "description": "Start and end page, or other equivalent indicator",
+                            "datatype": "xsd:string",
+                            "minCount": 1,
+                            "maxCount": 1
+                        },
+                        {
+                            "path": "http://doi.org/",
+                            "name": "DOI",
+                            "description": "Digital Object Identifier",
+                            "datatype": "xsd:string",
+                            "maxCount": 1
+                        },
+                        {
+                            "path": "https://www.ncbi.nlm.nih.gov/pmc/articles",
```

Comment:
The pubmed ID can be provided in the same way as the doi above using the [nexus/schemaorgsh/identifier/shapes/v0.1.0/identifiershape](https://github.com/INCF/neuroshapes/blob/master/modules/nexus-schemas/src/main/resources/schemas/nexus/schemaorgsh/identifier/v0.1.0.json).

Since multiple identifiers are needed (doi, pubmed id) it may be useful to create dedicated properties for them just like schema.org [did](http://schema.org/identifier). For example an "isbn" property is provided as an identifier sub-property. I've opened an [issue](https://github.com/INCF/neuroshapes/issues/49) for this.